bundler/spec/lock/lockfile_spec.rb

```diff
@@ -78,9 +79,6 @@
       DEPENDENCIES
         rack
 
-      CHECKSUMS
-        #{checksum_for_repo_gem(gem_repo2, "rack", "1.0.0")}
-
```

Comment:
Should this include the checksums section so that specs keep passing once generated by default?